lib/puppet/feature/base.rb

```diff
@@ -18,7 +18,7 @@
   begin
     # ruby
     require 'Win32API'          # case matters in this require!
-    require 'win32ole'
+    require 'win32ole' ; WIN32OLE.codepage = WIN32OLE::CP_UTF8
```

Comment:
Note added. Haven't found the significance of `WIN32OLE.locale` in this context - also noted.